src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/NonEmptyAtclauseDescriptionCheck.java

```diff
@@ -95,13 +95,111 @@
  * class Test
  * {
  * &#47;**
- * * Violation for param "b". Tags "deprecated", "throws" are ignored.
+ * * Violation for param "b".
  * * &#64;param a Some javadoc // OK
  * * &#64;param b
- * * &#64;deprecated
+ * *&#47;
+ * public int method(String a, int b)
+ * {
+ * return 1;
```

Comment:
please use proper indentation in javadoc to let it be read by humans easily even in javadoc format
https://github.com/checkstyle/checkstyle/blob/fe36c48a8b75bb97f5b4c617e0a79dfb724ead0a/src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java#L136-L139

Reply:
@romani  Like this
```
/**
  * Violation for param "b" and at tags "deprecated", "throws".
  * @param a Some javadoc // OK
  * @param b
  * @deprecated
  * @throws Exception
  */
public int method(String a, int b) throws Exception {
  return 1;
}
```

Reply:
And do it like this to all examples i added ?